### DISCUSSION TITLE
Zwift ride and Mywhoosh

### DISCUSSION THREAD

**Discussion by wattsup21** [cagnulein/qdomyos-zwift#2821] [Q&A]
Hello all,

First and foremost, the QZ app is genius and very thankful for its integration with many apps.

I have the Zwift ride and would like to use it with Mywhoosh and possibly Wahoo Systems but for now let's focus on Mywhoosh. I followed all the instructions to get the Zwift ride setup, connected and virtual gears to show in QZ and up/down. My 2 issues are below.

1) When I change gears, I do not feel any resistance change at all

2) My RPMS are very low (70rpm or below) and that is on gear zero/any gear I change it too (can't feel resistance when I change any ways), But I should be able to spin high RPMs like I do in Zwift, but I cannot. I Usually spin anywhere between 80-95rpms.

3) I use IOS (iPhone 11 max pro for QZ app) and MacBook for Mywhoosh.

As mentioned, Zwift ride is connected/shows in QZ app, virtual gears are connected (it turns from blinking blue to solid blue) and I see all other stats when riding to include gear changes, I just don't feel anything when I change gears and resistance seems strong for gear 0.

Any help appreciated. Thanks, in advanced!

**Comment by cagnulein** [comment]
Hi @wattsup21 thanks! I would like first of all to send you the latest beta.

send me an email to roberto.viola83 at gmail.com indicating this ticket and if you are on android or iOS. Thanks

**Reply by wattsup21** [reply]
Got it, I tried and was still have the same issue. Gear changing but no resistance. Let me try again tomorrow but I'm sure its something small I'm missing.

EDIT: I just tried it, did 4 miles. The gears worked and I believe I felt resistance, but I think my gears are all over the place. I went into the negative gear 0.00 -0.01, -.02, -.03 etc etc and got done to negative 3-5 range is where I felt the resistance ease up and I could pedal around 75-85rpms. But then, somehow the gears switched to my left gear which the gears up and the right switching them down and throughout the ride they switched back. I think I just need to fix the gears, and I should be OK, any tips?

**Reply by cagnulein** [reply]
@wattsup21 Could you create a debug log so I can check your setup? https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F

**Reply by wattsup21** [reply]
> @wattsup21 Could you create a debug log so I can check your setup? https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F

Just did it, let me know if you need anything else

**Reply by wattsup21** [reply]
Added log

**Comment by PMBreton** [comment]
Hi @cagnulein. Tagging on here as I have nearly the same set up/situation as @wattsup21. Trying to use Zwift Ride on MyWhoosh, with iPad for QZ and Windows laptop for mywhoosh. I'm struggling to get both the kickr and both controllers connected to QZ. Can you provide some advice/guidance?

**Reply by cagnulein** [reply]
hi @PMBreton did you follow this https://github.com/cagnulein/qdomyos-zwift/wiki/Virtual-Gearing-on-QZ-with-Zwift,-Rouvy,-Mywhoosh-and-Traning-Peaks#zwift-play-and-ride ?

**Reply by PMBreton** [reply]
I somehow missed that. Will give it another go. Thx!

**Reply by cagnulein** [reply]
ok let me know!
